### PR TITLE
#1016 BitbucketStars placeholder implementation.

### PR DESCRIPTION
**Pull request by criske**
pr for #1016

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

**Comment by zoeself** [comment]
@amihaiemil please review this Pull Request. Deadline (when it should be merged or closed) is ``2021-03-26T10:27:17.723398``.

You should check if the requirements have been implemented (partially or in full), if there are unit tests covering the changes and if the CI build passes. Feel free to reject the PR or ask for changes if it's too big or not clear enough.

Estimation here is ``30 minutes``, that's how much you will be paid. You will be paid even if this PR gets rejected.

**Comment by zoeself** [comment]
@amihaiemil Don't forget to close this ticket before the deadline (2021-03-26T10:27:18). You are past the first half of the allowed period.

File: self-core-impl/src/main/java/com/selfxdsd/core/BitbucketStars.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     * Logger.
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(
+            BitbucketStars.class
+    );
+
+    @Override
+    public boolean add() {
+        LOG.error("Bitbucket doesn't support repo starring.");
+        return false;
```
Comment (amihaiemil) [inline self-core-impl/src/main/java/com/selfxdsd/core/BitbucketStars.java:47]: @criske Pls log it with level ``warning`` and return ``true`` -- if it's not supported, we must assume the operation was successful.

**Review by amihaiemil — CHANGES_REQUESTED** [review]
@criske just a comment

**Comment by amihaiemil** [comment]
@rultor merge it please

**Comment by zoeself** [comment]
@amihaiemil thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Pull request merged**